tests/cursor/bug1050-002.phpt

```diff
@@ -0,0 +1,74 @@
+--TEST--
+PHPC-1050: Command cursor should not invoke getMore at execution (rewind omitted)
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php NEEDS('REPLICASET'); CLEANUP(REPLICASET); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(REPLICASET);
+
+$cmd = new MongoDB\Driver\Command(
+    [
+        'aggregate' => COLLECTION_NAME,
+        'pipeline' => [
+            ['$changeStream' => (object) []],
+        ],
+        'cursor' => (object) [],
+    ],
+    [
+        'maxAwaitTimeMS' => 1000,
+    ]
+);
+
+$start = microtime(true);
+$cursor = $manager->executeReadCommand(DATABASE_NAME, $cmd);
+printf("Executing command took %0.6f seconds\n", microtime(true) - $start);
+
+$it = new IteratorIterator($cursor);
+
+printf("Current position is valid: %s\n", $it->valid() ? 'yes' : 'no');
+
+$start = microtime(true);
+$it->next();
+printf("Advancing cursor took %0.6f seconds\n", microtime(true) - $start);
```

Comment:
Unlike the first test, this one omits the initial call to `rewind()` and tests that we preserve existing behavior by not requiring the rewind handler to ever be invoked (only relevant when wrapping a cursor with IteratorIterator).